app/views/evaluators/index.html.erb

```diff
@@ -2,39 +2,73 @@
   <div data-page="manage-evaluators">
   <%= render 'shared/back_link', path: phases_path(@phase) %>
     
-    <h1><%= @challenge.title %> - <%= @phase.title %></h1>
+    <h1><%= challenge_phase_title(@challenge, @phase) %></h1>
     <p>Create and manage a list of evaluators for the challenge.</p>
     <h2 class="padding-top-3">Add Evaluators</h2>
     <p>Evaluators will not be assigned to submissions until you assign in the submission detail view.</p>
-    <%= form_with(model: EvaluatorInvitation.new, url: phase_evaluators_path(@phase), method: :post, local: true) do |form| %>
+    <%= form_with(model: @evaluator_invitation || EvaluatorInvitation.new, url: phase_evaluators_path(@phase), method: :post, local: true, data: { controller: "evaluator-form modal" }, html: { novalidate: true }) do |form| %>
```

Comment:
the `evaluator-form` stimulus controller doesn't exist, did you mean to use `evaluation-form` controller?

Reply:
since you're using the `evaluation-form#validatePresence` function here, I think it would be good to extract that functionality into a separate controller to be used in both places. the rest of the EvaluationFormController related to the EvaluationForm fields can stay there. that way we'll have a reusable form validation controller for other stimulus forms.